clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -68,6 +68,15 @@
  * @see AggKafkaConsumerService which wraps one instance of this class per Kafka cluster.
  */
 public abstract class KafkaConsumerService extends AbstractKafkaConsumerService {
+  /**
+   * Waiting for the next poll() after unsubscribing, indicating that all previous inflight messages were processed
+   */
+  public static final int DEFAULT_WAIT_AFTER_UNSUBSCRIBE_TIMEOUT_MS = (int) TimeUnit.MINUTES.toMillis(5);
```

Comment:
ech.  I guess this is ok, but I think in most parts of the code we'd just write 300000 with a comment of // 5 minutes

Reply:
I don't know why you're so afraid of casts, hehe